torchci/rockset/utilization/__sql/runner_utilization_by_activity.sql

```diff
@@ -0,0 +1,28 @@
+SELECT
+  FORMAT_ISO8601(
+      DATE_TRUNC(
+          :granularity,
+          started_at AT TIME ZONE :timezone
+      )
+  ) AS started_date,
+  activity, 
+  SUM(DATE_DIFF('SECOND',started_at, completed_at)) AS duration,
+FROM (SELECT
+  PARSE_TIMESTAMP_ISO8601(started_at) as started_at,
+  PARSE_TIMESTAMP_ISO8601(completed_at) as completed_at,
+  IF(head_branch like 'ciflow/%',
+    CONCAT('ciflow/', ELEMENT_AT(SPLIT(head_branch, '/'), 2)),
+    IF(workflow_name = 'periodic', 'periodic', head_branch)) as activity
```

Comment:
Any comment about the meaning of the IF conditions here as I'm seeing both `ciflow/periodic` and `periodic` as the pseudo head branch on the preview.  So I'm trying to figure out what the latter means